native/cocos/engine/Engine.cpp

```diff
@@ -154,29 +157,18 @@ int Engine::restart() {
 }
 
 void Engine::close() { // NOLINT
-    if (cc::EventDispatcher::initialized()) {
-        cc::EventDispatcher::dispatchCloseEvent();
```

Comment:
Why delete this line ?
It's used for dispatching `onClose` event to JS.

Reply:
It is called before